sdk/tables/Azure.Data.Tables/tests/Azure.Data.Tables.Tests.csproj

```diff
@@ -8,12 +8,20 @@
     <PackageReference Include="nunit" />
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="Moq" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\identity\Azure.Identity\src\Azure.Identity.csproj" />
```

Comment:
Prefer a package reference to other libraries.

Reply:
In general, you'll want to go with a package reference.  There's a project variable embedded in the engineering system that allows you to request an override that makes all of your internal SDK references project-based for local testing  and such.   

I place this in a `Directory.Build.targets` file in my SDK directory when I want to use it:

```xml
<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <!--
      This area is for local development overrides; this targets files is not shared
      with the respository.
   -->
  <PropertyGroup>
    <UseProjectReferenceToAzureClients>true</UseProjectReferenceToAzureClients>
  </PropertyGroup>

  <!-- Import the common SDK build targets. -->
  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\.., Directory.Build.targets))\Directory.Build.targets" />

  <!-- Remind me about the overrides, in case I forget -->
  <Target Name="DisplayLocalOverrides" BeforeTargets="Build" Outputs="LocalOverrides">
      <Message Text="Local Override: UseProjectReferenceToAzureClients = $(UseProjectReferenceToAzureClients)" Importance="high"/>
  </Target>
</Project>
```

Reply:
Nice, thanks!